vllm/v1/engine/async_llm.py

```diff
@@ -398,7 +398,9 @@ async def start_profile(self) -> None:
     async def stop_profile(self) -> None:
         await self.engine_core.profile_async(False)
 
-    async def reset_prefix_cache(self) -> None:
+    async def reset_prefix_cache(self, device: Device) -> None:
+        if device == Device.CPU:
+            raise ValueError("Not supported on CPU.")
```

Comment:
IIRC we only support GPU, so may be just

```suggestion
        if device != Device.GPU:
            raise ValueError("Reset prefix cache only supports GPUs.")
```
@robertgshaw2-redhat @alexm-redhat could you help confirm TPU?